kuma/core/middleware.py

```diff
@@ -44,7 +54,7 @@ def process_request(self, request):
             if query_string:
                 full_path = '%s?%s' % (full_path, query_string)
 
-            response = HttpResponsePermanentRedirect(full_path)
+            response = HttpResponseRedirect(full_path)
```

Comment:
Please add this change to a different PR, against [bug 1385315](https://bugzilla.mozilla.org/show_bug.cgi?id=1385315). Also, it needs to apply to wiki pages and not any other page, so a middleware is probably not appropriate.